index.html

```diff
@@ -7,37 +7,192 @@
+    
     <link rel="stylesheet" href="./css/styles.css">
     <title>My Professional Work Portfolio</title>
 </head>
 <body>
-    <div class="header-container">
+   <section class="root-container">
+
+    <section class="header-section">
         <header>
             <nav class="nav-link">
                 <a href="#">My Logo</a>
                 <img src="./image/union-image.png" alt="Nav Bar" >
             </nav>    
         </header>    
         <section>
+            <div>
             <h1 class="developer-name">I’m Mariane Glad to see you!</h1>
             <p class="header-text">
                 I’m a software developer! I can help you build a product , 
                 feature or website Look through some of my work and experience! 
                 If you like what you see and have a project you need coded, 
                 don’t hestiate to contact me.
             </p>
+            </div>
             <div class="link-image">
                 <p class="connect-link">LET’S CONNECT</p>
             </div>
                 <ul class="connect-icon">
-                    <li><a href="#"><img src="./image/Linkedinicon.png" alt="" /></a></li>
+                    <li><a href="#"><img src="./image/Linkedinicon.png" alt="" id="image1" /></a></li>
                     <li><a href="#"><img src="./image/github.png" alt="" /></a></li>
                     <li><a href="#"><img src="./image/twitter.png" alt="" width="20" height="20" /></a></li>
                     <li><a href="#"><img src="./image/M.png" alt="" width="20" height="20" /></a></li>
                     <li><a href="#"><img src="./image/fingers.png" alt="" width="20" height="20" /></a></li>
                 </ul>        
         </section>
-    </div>
+    </section>
+
+    <section class="tonic-card">
+      <div class="container-image-1">
+      </div>
+        <section class="left-block">                
+            <div class="tonic-title">
+                    <h2>Tonic</h2>
+                <ul class="frame-2">
+                    <li class="canopy-text"><h3>CANOPY</h3></li>
+                    <li class="role-text"><h3>Back End Dev</h3></li>
+                    <li class="year-text"><h3>2015</h3></li>
+                </ul>
+            </div>
+            <div class="text-paragraph">
+                <p>
+                    A daily selection of privately personalized reads; 
+                    no accounts or sign-ups required.
+                </p>
+            </div> 
+            
+            <ul class="primary-buttons">
+                <li><button class="button1" type="button">html</button></li>
+                <li><button class="button2" type="button">css</button></li>
+                <li><button class="button3" type="button">Javascript</button></li>
+            </ul>
+                    <button class="see-project" type="button">See Project</button>
+        </section>
+    </section>
+    
+    <section class="tonic-card">

```

Comment:
### Figma Template
 ![missing-margin-background (copy)](https://user-images.githubusercontent.com/93039359/177990025-c1ba578f-3e72-4c9c-ae7f-053fa8d1eac1.png)

### Your Project

![figma](https://user-images.githubusercontent.com/6615394/177887210-c66a42d0-625e-423b-8d85-71ffcb26d023.png)

- [ ] You are missing these margins and paddings shown in the green box. Please add it to your design. :blush: